src/sentry/nodestore/bigtable/backend.py

```diff
@@ -97,22 +99,31 @@ def connection(self):
         return get_connection(self.project, self.instance, self.table, self.options)
 
     def get(self, id):
+        item_from_cache = self._get_cache_item(id)
+        if item_from_cache:
+            return item_from_cache
         return self.decode_row(self.connection.read_row(id))
```

Comment:
Well, it's somewhat intended for now, in that I'm not convinced it's a problem yet, But it might be and I'd like to check the stats about this before adding the feature.

Reply:
But you are setting the values back to cache in get_multi. Let's at least be consistent. Also what is the planned usage for this cache? Is it meant to be a standard LRU cache independently of the usage the user does of it. Or is it optimized to be effective for events reused after a node is saved .

Right now you would be caching elements only through get_multi and if the cache_on_save option is checked.

Reply:
Oops, yes there is a missing set cache here. Yes, we are looking to cache any items that have been accessed recently (with the cache_on_save as a separate option so we can experiment with caching every event as it is saved to sentry)